go.mod

```diff
@@ -63,6 +63,7 @@ require (
 	golang.org/x/crypto v0.21.0
 	golang.org/x/net v0.21.0
 	golang.org/x/oauth2 v0.17.0
+	golang.org/x/text v0.14.0
 	google.golang.org/api v0.150.0
```

Comment:
Why is go.sum not updated?

Reply:
golang.org/x/text v0.14.0 was already in the go.sum file.